cms/server/admin/templates/task.html

```diff
@@ -426,11 +426,46 @@ <h4 id="title_task_type_{{ dataset.id }}" class="toggling_on">Dataset configurat
 
   <!-- Test cases for this dataset -->
 
+  <script type="text/javascript">
+      selectedTestcases = [];
+      function addSelectedTestcases(testcase_id)
+      {
+          for (var i = 0; i < selectedTestcases.length; i++)
+              if (selectedTestcases[i] == testcase_id)
+              {
+                  selectedTestcases[i] = -1;
+                  return;
+              }
+          selectedTestcases[selectedTestcases.length] = testcase_id;
+      }
+
+      function sendDeleteTestcases()
+      {
+          var sel = [];
+          for (var i = 0; i < selectedTestcases.length; i++)
+              if (selectedTestcases[i] != -1)
+                  sel[sel.length] = selectedTestcases[i];
+          if (sel.length < 1)
+          {
+              alert("Please select at least one testcase to delete!");
+              return;
+          }
+          var str = "";
+          str += sel[0];
+          for (var i = 1; i < sel.length; i++)
+              str += '-' + sel[i];
+          CMS.AWSUtils.ajax_delete('{{ url_root }}/dataset/{{ dataset.id }}/testcase/' + str + '/delete');
```

Comment:
I think it's better to send the testcases as the data rather than in the URL. Like posting forms. 
The url scheme is a bit ugly this way.